.github/workflows/manual.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           git clone -b ${{ env.BESS_DPDK_BRANCH }} https://github.com/omec-project/bess.git
           cd bess/env
           yes n | ./rebuild_images.py focal64
-          docker tag omec-project/bess_build:latest ${{ env.REPO }}:${{ env.TAG }}
+          docker tag nefelinetworks/bess_build:latest ${{ env.REPO }}:${{ env.TAG }}
```

Comment:
Looks like `docker tag` line is not required. The command ` ./rebuild_images.py focal64` builds the BESS docker image `ghcr.io/omec-project/upf-epc/bess_build:latest`. So we just need to push the built BESS docker image which is already taken care of

Reply:
> Looks like `docker tag` line is not required. The command ` ./rebuild_images.py focal64` builds the BESS docker image `ghcr.io/omec-project/upf-epc/bess_build:latest`. So we just need to push the built BESS docker image which is already taken care of

Good point, however, I am wondering whether the docker tag is being used somewhere else (e.g., AiaB, through the helm charts). Would not the safe option to just revert the change (as indicated in the patch) and later on remove the `tag` after confirming that is not being use in any of our other projects?